Headers/DebugServer2/Core/CPUTypes.h

```diff
@@ -55,6 +55,10 @@ enum CPUType {
   // skip 17
   kCPUTypePOWERPC = 18,
   kCPUTypePOWERPC64 = (kCPUTypePOWERPC | kCPUArchABI64),
+  // skip 19
+  kCPUTypeRISCV32 = 0x5032,
```

Comment:
Hardcoded values -- maybe mention / reference where these came from?

Reply:
This is a bit more difficult ... these are effectively placeholders.  The rest of these values are from libSystem from Apple.  However, Apple does not yet have constants codified for RVnnn.  I used these values as placeholders, though they should match PE/COFF's specification (because I knew that off hand).  I can add a reference and a TODO, that sounds like a good idea.